src/ClientConfig_TEST.cc

```diff
@@ -76,7 +76,7 @@ std::string cachePath()
 TEST(ClientConfig, InitiallyDefaultServers)
 {
   ClientConfig config;
-  EXPECT_EQ(1u, config.Servers().size());
+  EXPECT_EQ(2u, config.Servers().size());
```

Comment:
We should add a note to the migration guide, since this is a slight behaviour change.

Reply:
Added in [352f42d](https://github.com/gazebosim/gz-fuel-tools/pull/263/commits/352f42d3ea70743024145517f37203f23b68259f)